demo/image_mod_default_image/run.py

```diff
@@ -0,0 +1,18 @@
+import gradio as gr
```

Comment:
Adding another demo that sets the default value on the image since there wasn't an example doing that already and that was mentioned on the discord.